backend/src/main/java/com/redhat/cloud/notifications/routers/EndpointResource.java

```diff
@@ -378,7 +396,11 @@ public Response updateEndpoint(@Context SecurityContext sec,
                 databaseEndpointProps.setBasicAuthentication(incomingEndpointProps.getBasicAuthentication());
                 databaseEndpointProps.setSecretToken(incomingEndpointProps.getSecretToken());
 
-                this.secretUtils.updateSecretsForEndpoint(dbEndpoint);
+                try {
+                    this.secretUtils.updateSecretsForEndpoint(dbEndpoint);
+                } catch (final SourcesException e) {
+                    Log.error(e.getMessage(), e);
+                }
```

Comment:
Like for the endpoint creation, I would be in favor of a failure (500) returned to the caller if an endpoint update fails because we were unable to update a secret in Sources. The error log entry is still needed.

Reply:
Agree, yes. I will return a similar message to the one above.

Reply:
Done!